### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ccheney**
☀️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+5. Open your browser's console to see the HMR logs
+6. Modify any files in `src`
+7. ???
+8. In your shell, watch webpack rebuild the example application
+9. In your browser's console, watch webpack rebuild and update modules
+10. $$$
 
-### Tasks
+## dev tasks
+- $ `yarn dev` (local development)
```
Comment (codeBelt) [inline README.md]: Should probably say (local development with hapi server)

File: README.md [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+- $ `yarn lint` (eslint)
 
-`yarn dev` (Development)
-
-`yarn build` (Production) or with server `yarn start`
+## build tasks
+- $ `yarn start` (hapi server)
```
Comment (codeBelt) [inline README.md]: should probably say (production build with hapi server)

File: README.md [hunk 3]
```diff
@@ -1,9 +1,22 @@
-# hapi-react-hot-loader (Work In Progress)
+# `hapi-react-hot-loader`
```
Comment (codeBelt) [inline README.md]: Where is my Never do `yarn clean`? I found that helpful

File: README.md [hunk 4]
```diff
@@ -1,9 +1,22 @@
...
+Example application demonstrating hot-module-reloading with webpack2, react and friends, and hapi.
 
-Never do `yarn clean`.
+## get started
+1. Clone this repo
```
Comment (codeBelt) [inline README.md]: 1.5 Star this repo

File: README.md [hunk 5]
```diff
@@ -1,9 +1,22 @@
...
+2. $ `yarn`
+3. $ `yarn dev`
+4. http://localhost:3000
+5. Open your browser's console to see the HMR logs
```
Comment (codeBelt) [inline README.md]: Do we need to tell the person this?

File: README.md [hunk 6]
```diff
@@ -1,9 +1,22 @@
...
+6. Modify any files in `src`
+7. ???
+8. In your shell, watch webpack rebuild the example application
```
Comment (codeBelt) [inline README.md]: Do we need to tell the person this?

File: README.md [hunk 7]
```diff
@@ -1,9 +1,22 @@
...
+9. In your browser's console, watch webpack rebuild and update modules
```
Comment (codeBelt) [inline README.md]: Do we need to tell the person this?

File: README.md [hunk 8]
```diff
@@ -1,9 +1,22 @@
-# hapi-react-hot-loader (Work In Progress)
+# `hapi-react-hot-loader`
+Example application demonstrating hot-module-reloading with webpack2, react and friends, and hapi.
```
Comment (codeBelt) [inline README.md]: should say "hot-module-reloading 3"

File: README.md (hunk 3 above)
Comment (ccheney) [inline README.md]: `yarn clean` isn't relevant to the project

File: README.md (hunk 5 above)
Comment (ccheney) [reply README.md]: Yeah, it's the whole point of this example

File: README.md (hunk 6 above)
Comment (ccheney) [reply README.md]: Yes

File: README.md (hunk 7 above)
Comment (ccheney) [reply README.md]: Yes

**Pull request merged**